wgpu-hal/src/gles/adapter.rs

```diff
@@ -450,6 +450,12 @@ impl super::Adapter {
             supported((3, 1), (4, 2)) || extensions.contains("GL_ARB_shader_image_load_store"),
         );
         features.set(wgt::Features::SHADER_UNUSED_VERTEX_OUTPUT, true);
+        let query_buffers = extensions.contains("GL_ARB_query_buffer_object")
+            || extensions.contains("GL_AMD_query_buffer_object");
+        if extensions.contains("GL_ARB_timer_query") && query_buffers {
```

Comment:
Does the code as written work with GLES's `EXT_disjoint_timer_query`/`EXT_disjoint_timer_query_webgl2`? I suspect it might?

Reply:
Probably. They'd need `glow` support. Not sure if I can test those either.

Reply:
Alright, we can leave those out.